src/words/Letter.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {isNull} from 'lodash';
+
+function setColor(correct) {
+  if (isNull(correct)) {
+    return 'black'
+  }
+
+  return correct ? 'green' : 'red'
+}
+
+function Letter(props) {
```

Comment:
Instead of using `content`, use `children` instead.

```javascript
function Letter(props) {
.
.
.
return <span style={style}>
      {children}
 </span>
}

// call Letter
<Letter>something</Letter>


```

Reply:
how about the `isCorrect` .how should it be passed?

Reply:
Normally via `props`